src/js/libgeo/GeoOps.js

```diff
@@ -2589,6 +2589,107 @@ geoOps.TransformPolygon.updatePosition = function(el) {
     }));
 };
 
+geoOps.TrComposeTrTr = {};
+geoOps.TrComposeTrTr.kind = "Tr";
+geoOps.TrComposeTrTr.signature = ["Tr", "Tr"];
+geoOps.TrComposeTrTr.updatePosition = function(el) {
+    var a = csgeo.csnames[el.args[0]];
+    var b = csgeo.csnames[el.args[1]];
+    el.matrix = List.normalizeMax(List.productMM(b.matrix, a.matrix));
+    el.dualMatrix = List.normalizeMax(List.productMM(b.dualMatrix, a.dualMatrix));
+};
+
+geoOps._helper.composeMtMt = function(el, m, n) {
```

Comment:
Let's see. The best way to think about this is probably in terms of matrices `[[x, -y], [y, x]]` representing our doubly complex numbers. In this setup, a Möbius transformation is a 4×4 matrix. I computed the product of these, and tried to capture the formulas for the individual components in a short formula. There is also the matter of conjugation: if the second operation (left factor, `n`) is an anti-Möbius transformation, then the first transformation (right factor, `m`) has to be conjugated (replacing all `y` by `-y` in the doubly complex representation) before the matrices are multiplied. That's what the `n.anti ? … : …` constructs are all about. `addsub` does add in the direct and subtract in the reversing situation, for `subadd` it's the other way round. Then I collect all those coordinates, normalize them together before constructing a Möbius transformation from them. Does this help? How can this be made clearer, short of pasting the whole output from Sage as a comment to explain the matrix multiplication?